hedera-node/hedera-mono-service/src/main/java/com/hedera/services/fees/calculation/schedule/txns/ScheduleCreateResourceUsage.java

```diff
@@ -31,6 +36,7 @@
 public class ScheduleCreateResourceUsage implements TxnResourceUsageEstimator {
     private final ScheduleOpsUsage scheduleOpsUsage;
     private final GlobalDynamicProperties dynamicProperties;
+    private long lifetimeSecs;
```

Comment:
⚠️ This class must be thread-safe, because it can be used concurrently from gRPC threads **and `handleTransaction`**. So we need to re-compute this value within both `usageGiven()` and `secondaryFeesFor()`.

(Which should be no big deal, since it's a very fast method. 😇 )

Reply:
Ah yes .. fixed . Thanks ! 🙏

Reply:
Added a javadoc for the class , so I don't forget again !